src/components/utils/test/helper.cc

```diff
@@ -35,22 +35,25 @@ namespace test {
 namespace components {
 namespace utils_test {
 
+const int_least32_t kRetryPeriodMs = 100;
+
 MainAndWorkerThreadSynchronisation::MainAndWorkerThreadSynchronisation()
     : done_cond_()
     , done_lock_()
     , done_auto_lock_(done_lock_)
     , var_lock_()
     , job_is_done(false) {}
 
-void MainAndWorkerThreadSynchronisation::WaitJobDone(const uint32_t milliseconds) {
-  while (true) {
+void MainAndWorkerThreadSynchronisation::WaitJobDone(int_least32_t miliseconds) {
```

Comment:
@LuxoftAKutsan, fixed in [1fb7d45](https://github.com/smartdevicelink/sdl_core/pull/707/commits/1fb7d45870bfeab6926d2d7da6857a36028bdb0b).